kin-sdk-core/src/androidTest/java/kin/sdk/core/KinAccountTest.java

```diff
@@ -145,6 +145,48 @@ public void sendTransactionSync_wrongPassphrase() {
         }
     }
 
+    @Test
+    public void sendTransactionSync_SecondTimeEmptyPassphraseFails() throws Exception {
+        KinAccount senderAccount = importedAccounts.get(0);
+        Balance senderBalance = senderAccount.getBalanceSync();
+        BigDecimal amountToSend = new BigDecimal(10);
+
+        senderAccount.sendTransactionSync(kinAccount.getPublicAddress(), PASSPHRASE, amountToSend);
+
+        Balance kinAccountBalance = kinAccount.getBalanceSync();
+        assertTrue(kinAccountBalance.value(0).equals("10"));
+
+        Balance afterBalance = senderAccount.getBalanceSync();
+        assertTrue((senderBalance.value().subtract(amountToSend).compareTo(afterBalance.value())) == 0);
+
+        try {
+            senderAccount.sendTransactionSync(kinAccount.getPublicAddress(), "", new BigDecimal(1));
+        } catch (Exception e) {
+            assertEquals("Wrong passphrase - could not decrypt key with given passphrase", e.getCause().getMessage());
```

Comment:
we need to verify the exception is indeed thrown 
same for the second test below